src/styles/menu.scss

```diff
@@ -164,7 +164,7 @@ $block: #{$fd-namespace}-menu;
       }
 
       @include fd-focus() {
-        outline-color: var(--sapContent_ContrastFocusColor);
+        outline-color: var(--fdMenuActiveStateFocus);
```

Comment:
From what I can see from other variable names, we use camelCase for defining the component name, and then we use `_` as a separator for the property name, so it should be like `fdMenu_ActiveState_Focus`

Reply:
fixed